src/Nest/XPack/RollUp/DeleteRollupJob/DeleteRollupJobResponse.cs

```diff
@@ -5,5 +5,12 @@ namespace Nest
 	[JsonObject]
 	public interface IDeleteRollupJobResponse : IAcknowledgedResponse { }
 
-	public class DeleteRollupJobResponse : AcknowledgedResponseBase, IDeleteRollupJobResponse { }
+	public class DeleteRollupJobResponse : AcknowledgedResponseBase, IDeleteRollupJobResponse
+	{
+		/// <summary>
+		/// Checks whether the response returned a valid HTTP status code and that the delete is acknowledged
+		/// in one go. See also <see cref="AcknowledgedResponseBase.Acknowledged"/>
+		/// </summary>
+		public override bool IsValid => base.IsValid && Acknowledged;
```

Comment:
I will remove it from `6.x` but really want to move this check to the base in `7.x`

Reply:
Rerunning the test I can see I added this to make it behave the same as other CRUD related operations.

The following tests fails if I remove this check. 

```
    Tests.XPack.Rollup.RollupJobCrudTests.DeleteNotFoundIsNotValid [FAIL]
      Expected False because Valid NEST response built from a successful low level call on DELETE: /_xpack/rollup/job/rollup-logs-fluent-61256519-r
ollupjob?pretty=true&error_trace=true
      # Audit trail of this API call:
       - [1] HealthyResponse: Node: https://localhost:9200/ Took: 00:00:00.0009643
      # Request:
      <Request stream not captured or already read to completion by serializer. Set DisableDirectStreaming() on ConnectionSettings to force it to b
e set on the response.>
      # Response:
      {
  "acknowledged" : false
}
```

Merging it with this included.